bats/sql.bats

```diff
@@ -48,6 +48,23 @@ teardown() {
     [ "${#lines[@]}" -eq 8 ]
 }
 
+@test "save query" {
```

Comment:
Maybe this should be it's own bats test file?

Reply:
would be nice to see a conflict test in bats also.

Reply:
Done.